### PR TITLE
Increased timeout managed folder e2e tests

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Increased timeout for managed folder e2e tests as it is intermittently failing.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: tools/integration_tests/managed_folders/admin_permissions_test.go [hunk 1]
```diff
@@ -60,8 +60,8 @@ func (s *managedFoldersAdminPermission) Setup(t *testing.T) {
 	if s.managedFoldersPermission != "nil" {
 		providePermissionToManagedFolder(bucket, path.Join(testDir, ManagedFolder1), serviceAccount, s.managedFoldersPermission, t)
 		providePermissionToManagedFolder(bucket, path.Join(testDir, ManagedFolder2), serviceAccount, s.managedFoldersPermission, t)
-		// Waiting for 10 seconds as it usually takes 10 seconds for policy changes to propagate.
-		time.Sleep(10 * time.Second)
+		// Waiting for 18 seconds for policy changes to propagate.
```
Comment (sethiay) [inline tools/integration_tests/managed_folders/admin_permissions_test.go:63]: just curious - why 18 and not just 20 ?

Comment (Tulsishah) [reply tools/integration_tests/managed_folders/admin_permissions_test.go:63]: Thought to keep as less as possible. 15s worked but on louhi one of the OS it failed. That's why increased it to 18s further.

Comment (sethiay) [reply tools/integration_tests/managed_folders/admin_permissions_test.go:63]: let's make it 20s (2s buffer) and write that in a comment that this value is based on experiment.

Comment (Tulsishah) [reply tools/integration_tests/managed_folders/admin_permissions_test.go]: Done

**Review by sethiay — APPROVED** [review]

**Pull request merged**